clash-cores/src/Clash/Cores/LatticeSemi/ECP5/Blackboxes/IO.hs

```diff
@@ -0,0 +1,71 @@
+{-|
+  Copyright   :  (C) 2020, Foamspace corp
```

Comment:
Is this correct or a copy-paste error? :)

Reply:
See the latest commit.